core/pom.xml

```diff
@@ -90,6 +90,11 @@
             <artifactId>guava</artifactId>
             <version>25.1-jre</version>
         </dependency>
+        <dependency>
```

Comment:
Why do we need to add a new dependency for this?

Reply:
It's the commons-lang3 module that has been added, since it's used by the cvs reader. OSGi runs independent modules (bundles) in isolation, and the core OSGi bundle doesn't depend on any other bundle. Without this dependency the core bundle missed a class (I don't remember which) from commons-lang3. Presumably it's also required by some other (optional) module, so you won't notice when run in a normal runtime.

When we make the modules more independent, each must list its own dependencies to make sure that each bundle's dependencies are satisfied (considering that all modules depend on the core, but not the other way around).

Reply:
Good catch. I just pushed a change to `master` so that we no longer depend on `commons-lang3` anymore in an effort to keep the core as small as possible, so you should be able to remove this now